packages/cli/src/commands/contract/test.ts

```diff
@@ -25,13 +25,13 @@ export class CompileContract extends Command {
   static args = {
     contractName: Args.string({
       name: "contractName",
-      required: true,
+      default: "",
```

Comment:
Is there a reason for having a default empty string instead of undefined?

Reply:
because you cannot put undefined into string array https://github.com/AstarNetwork/swanky-cli/blob/0afb6a1232258c67c7dd27d2a657a27eb41a9478/packages/cli/src/commands/contract/test.ts#L50

Reply:
Even though oclif Arg default is set to empty string "", args.contractName is undefined. I'm not sure why this behaviour